### PR TITLE
Work on lambda functions

### PR DESCRIPTION
**Pull request by Chufle**
file: ./photo_checkin/src/photo_checkin.py
I placed the functions I need for the process in the py file. Next step write the functions

### CODE REVIEW

File: photo-checkin/src/photo_checkin.py [hunk 1]
```diff
@@ -1,20 +1,16 @@
-import urllib.parse
-import boto3
+def get_s3_object(event):
+    bucket = ()
+    file_name = ()
+    creation_date = ()
+    return bucket, file_name, creation_date
```
Comment (fabianschmauder) [inline photo-checkin/src/photo_checkin.py:5]: ? This is not ready or?

Comment (Chufle) [reply photo-checkin/src/photo_checkin.py:5]: no :)

Comment (Chufle) [reply photo-checkin/src/photo_checkin.py:5]: in next pull requests

**Comment by Chufle** [comment]
pull request obsolete. Check following pull request: "write-lambda-function-get-s3-object"

**Pull request closed without merging**